tests/test_scorecard.py

```diff
@@ -0,0 +1,14 @@
+# tests/test_scorecard.py
+import unittest
+from src.ossf_scorecard.scorecard import get_scorecard
+
+class TestScorecard(unittest.TestCase):
```

Comment:
Let's do the tests on a couple more diverse repos so we make sure we have variety of data and our models work fine in all cases.